src/Microsoft.Tye.Core/ProjectReader.cs

```diff
@@ -268,6 +268,15 @@ private static void EvaluateProject(OutputContext output, ProjectServiceBuilder
             project.Frameworks.AddRange(sharedFrameworks.Select(s => new Framework(s)));
             output.WriteDebugLine($"Found shared frameworks: {string.Join(", ", sharedFrameworks)}");
 
+            // determine container base image
+            if (project.ContainerInfo == null)
```

Comment:
Why is this here?

Reply:
See @jkotalik comment : https://github.com/dotnet/tye/pull/447#discussion_r422245910

Reply:
It's also set here https://github.com/dotnet/tye/pull/447/files#diff-b637c96b54d57dc51cd9e2aaca55198fR101

Reply:
@davidfowl I agree and it appears that it is the only entry point. However, @jkotalik is suggesting that there should be a null check, if it is (although it shoudn't be) we must create the instance.

previously I was just setting the values - `project.ContainerInfo!.BaseImageName`